README.md

```diff
@@ -134,29 +134,56 @@ faked in the browser by rebuilding the native UI using DOM elements.
 
 ## Building the Webapp
 
-First, you must create a `config.local.json` file in the root of your project
-with some configuration information. Specifically, it needs the location of a
-[source checkout of Atom](https://github.com/atom/atom) that has been built at
-the revision specified in the `config.json` file.
+#### 1. Checkout this project
 
+```sh
+git clone git@github.com:facebooknuclide/atom-in-orbit.git
 ```
-{
-  "ATOM_SRC": "/home/mbolin/code/atom"
-}
+
+#### 2. Checkout [Atom](https://github.com/atom/atom)'s source
+
+```sh
+git clone git@github.com:atom/atom.git
 ```
 
-You can build the local demo by running (this takes 10s on my Linux box):
+#### 3. Reset Atom to the revision in [`config.json`](https://github.com/facebooknuclide/atom-in-orbit/blob/master/config.json)
 
+```sh
+cd atom;
+git reset --hard `grep 'ATOM_REVISION' ../atom-in-orbit/config.json | sed 's/[^0-9a-f]//g'`
 ```
-$ npm run build
+
+#### 4. Build Atom
+
+```sh
+scripts/build;
```

Comment:
This should be ``script/build``, although this instruction should not be included as it is a part of atoms build instructions and not atom-in-orbit, this instruction alone does not suffice to build atom. The link you added below includes all steps to build atom and should suffice for step 4.